container_pipeline/workers/delivery.py

```diff
@@ -55,7 +56,8 @@ def deliver_build(self, job):
             if not delivery_id:
                 return False
         finally:
-            self.openshift.delete_pods(project, delivery_id)
+            time.sleep(50)
```

Comment:
Why are we adding this 50 seconds delay ? because openshift will refuse to delete it immediately ? (please add comment there as well)